src/main/java/core/basesyntax/FigureSupplier.java

```diff
@@ -0,0 +1,30 @@
+package core.basesyntax;
+
+public class FigureSupplier {
+    public static final int RANDOM_LIMIT = 342;
+
+    public Figure getRandomFigure() {
+        ColorSupplier colorSupplier = new ColorSupplier();
+        int figureIndex = (int) (Math.random() * 5);
```

Comment:
5 is a magic number - move it to constant as well

Reply:
done